import os
from subprocess import call
from subprocess import Popen
import subprocess
from subprocess import sys
from selenium.webdriver.common.by import By
# pip install selenium
from selenium import webdriver
from selenium.webdriver.firefox.options import Options
from selenium.common.exceptions import NoSuchElementException
import gspread

try:
   
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/Rozetka_Parcer/Rozetka_Parcer/Rozetka_Parcer.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/ALLOParcer/ALLOParcer/ALLOParcer.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/ParcerALLO no marketplace/ParcerALLO no marketplace/ParcerALLO_no_marketplace.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/ParcerALLO_no_marcetplace_2_magnitol/ParcerALLO_no_marcetplace_2_magnitol/ParcerALLO_no_marcetplace_2_magnitol.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/ParcerFoxtrot/ParcerFoxtrot/ParcerFoxtrot.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/parcerEpicentr/parcerEpicentr/ParcerATL.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/ParcerAUTOBAZA/ParcerAUTOBAZA/ParcerAUTOBAZA.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/ParcerWinautoVersion2.0/ParcerWinautoVersion2.0/ParcerWinautoVersion2.0.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/ParcerNotebooker/ParcerNotebooker/ParcerNotebooker.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/ParcerCitrus/ParcerCitrus/ParcerCitrus.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/Parcer_ITBOX_Brain/Parcer_ITBOX_Brain/Parcer_ITBOX_Brain.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/ParcerCitrus/ParcerCitrus/ParcerCitrus.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/ParcerEldorado/ParcerEldorado/ParcerEldorado.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/ParcerATL_ATL/ParcerATL_ATL/ParcerATL_ATL.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/TTT_Parcer/TTT_Parcer/TTT_Parcer.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/Stylus_Parcer/Stylus_Parcer/Stylus_Parcer.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/ZZHUK_parcer/ZZHUK_parcer/ZZHUK_parcer.py'])
except Exception as e:
    print ('Process failed: ',e)
try:
    subprocess.call([r'C:\Users\Konstantin\Documents\Visual Studio 2019\Python Scripts\Python 3.7 (64-bit)\python.exe','C:/Users/Konstantin/source/repos/ParcerMTI/ParcerMTI/ParcerMTI.py'])
except Exception as e:
    print ('Process failed: ',e)